### DISCUSSION TITLE
BE因为auto_compact 崩溃，无法重启，取消自动compact才能启动成功

### DISCUSSION THREAD

**Discussion by lhjw9810** [apache/doris#31082] [RFC]
`SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/be/data5/doris/be/lib/java_extensions/preload-extensions/preload-extensions-jar-with-dependencies.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/be/data5/doris/be/lib/java_extensions/java-udf/java-udf-jar-with-dependencies.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/be/data5/doris/be/lib/hadoop_hdfs/common/lib/slf4j-reload4j-1.7.36.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Reload4jLoggerFactory]
terminate called after throwing an instance of 'std::length_error'
  what():  basic_string::_M_create
*** Query id: 0-0 ***
*** tablet id: 84551 ***
*** Aborted at 1708306097 (unix time) try "date -d @1708306097" if you are using GNU date ***
*** Current BE git commitID: 003a815b63 ***
*** SIGABRT unknown detail explain (@0xc228) received by PID 49704 (TID 51319 OR 0x7f52873df700) from PID 49704; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /home/zcp/repo_center/doris_release/doris/be/src/common/signal_handler.h:417
 1# 0x00007F559DC3D270 in /lib64/libc.so.6
 2# raise in /lib64/libc.so.6
 3# abort in /lib64/libc.so.6
 4# __gnu_cxx::__verbose_terminate_handler() [clone .cold] at ../../../../libstdc++-v3/libsupc++/vterminate.cc:75
 5# __cxxabiv1::__terminate(void (*)()) at ../../../../libstdc++-v3/libsupc++/eh_terminate.cc:48
 6# 0x00005586BD36DAD1 in /home/be/data5/doris/be/lib/doris_be
 7# 0x00005586BD36DC24 in /home/be/data5/doris/be/lib/doris_be
 8# std::__throw_length_error(char const*) at ../../../../../libstdc++-v3/src/c++11/functexcept.cc:82
 9# doris::segment_v2::InvertedIndexColumnWriterImpl<(doris::FieldType)26>::add_array_values(unsigned long, void const*, unsigned char const*, unsigned char const*, unsigned long) at /home/zcp/repo_center/doris_release/doris/be/src/olap/rowset/segment_v2/inverted_index_writer.cpp:354
10# doris::segment_v2::ArrayColumnWriter::append_data(unsigned char const**, unsigned long) at /home/zcp/repo_center/doris_release/doris/be/src/olap/rowset/segment_v2/column_writer.cpp:965
11# doris::segment_v2::ArrayColumnWriter::append_nullable(unsigned char const*, unsigned char const**, unsigned long) at /home/zcp/repo_center/doris_release/doris/be/src/olap/rowset/segment_v2/column_writer.cpp:983
12# doris::segment_v2::ColumnWriter::append(unsigned char const*, void const*, unsigned long) in /home/be/data5/doris/be/lib/doris_be
13# doris::segment_v2::SegmentWriter::append_block(doris::vectorized::Block const*, unsigned long, unsigned long) in /home/be/data5/doris/be/lib/doris_be
14# doris::VerticalBetaRowsetWriter::add_columns(doris::vectorized::Block const*, std::vector<unsigned int, std::allocator<unsigned int> > const&, bool, unsigned int) at /home/zcp/repo_center/doris_release/doris/be/src/olap/rowset/vertical_beta_rowset_writer.cpp:123
15# doris::Merger::vertical_compact_one_group(std::shared_ptr<doris::Tablet>, doris::ReaderType, std::shared_ptr<doris::TabletSchema>, bool, std::vector<unsigned int, std::allocator<unsigned int> > const&, doris::vectorized::RowSourcesBuffer*, std::vector<std::shared_ptr<doris::RowsetReader>, std::allocator<std::shared_ptr<doris::RowsetReader> > > const&, doris::RowsetWriter*, long, doris::Merger::Statistics*) at /home/zcp/repo_center/doris_release/doris/be/src/olap/merger.cpp:245
16# doris::Merger::vertical_merge_rowsets(std::shared_ptr<doris::Tablet>, doris::ReaderType, std::shared_ptr<doris::TabletSchema>, std::vector<std::shared_ptr<doris::RowsetReader>, std::allocator<std::shared_ptr<doris::RowsetReader> > > const&, doris::RowsetWriter*, long, doris::Merger::Statistics*) at /home/zcp/repo_center/doris_release/doris/be/src/olap/merger.cpp:349
17# doris::Compaction::do_compaction_impl(long) at /home/zcp/repo_center/doris_release/doris/be/src/olap/compaction.cpp:354
18# doris::Compaction::do_compaction(long) at /home/zcp/repo_center/doris_release/doris/be/src/olap/compaction.cpp:125
19# doris::CumulativeCompaction::execute_compact_impl() at /home/zcp/repo_center/doris_release/doris/be/src/olap/cumulative_compaction.cpp:79
20# doris::Compaction::execute_compact() at /home/zcp/repo_center/doris_release/doris/be/src/olap/compaction.cpp:107
21# doris::Tablet::execute_compaction(doris::Compaction&) at /home/zcp/repo_center/doris_release/doris/be/src/olap/tablet.cpp:1910
22# std::_Function_handler<void (), doris::StorageEngine::_submit_compaction_task(std::shared_ptr<doris::Tablet>, doris::CompactionType, bool)::$_0>::_M_invoke(std::_Any_data const&) at /var/local/ldb_toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/std_function.h:291
23# doris::ThreadPool::dispatch_thread() in /home/be/data5/doris/be/lib/doris_be
24# doris::Thread::supervise_thread(void*) at /home/zcp/repo_center/doris_release/doris/be/src/util/thread.cpp:499
25# start_thread in /lib64/libpthread.so.0
26# __clone in /lib64/libc.so.6`

**Comment by zhbdesign** [comment]
这个怎么解决的
